test/falco_tests.yaml

```diff
@@ -20,22 +20,55 @@ trace_files: !mux
   builtin_rules_no_warnings:
     detect: False
     trace_file: trace_files/empty.scap
-    rules_warning: False
 
+  # The rules_events part of this test was mistakenly disabled as a
+  # part of the lua to c++ rules switch. The rule loading
```

Comment:
For context, this regression comes from way before the C++ porting (removal of `sinsp_rule_utils.lua`): https://github.com/falcosecurity/falco/pull/1715

Reply:
Oh, ok! I will update the comment.